cupy/__init__.py

```diff
@@ -309,6 +309,10 @@ def is_available():
 from cupy._manipulation.rearrange import roll  # NOQA
 from cupy._manipulation.rearrange import rot90  # NOQA
 
+# Borrowed from NumPy
+if hasattr(_numpy, 'broadcast_shapes'):  # NumPy 1.20
+    from numpy import broadcast_shapes  # NOQA
```

Comment:
We have `cupy._core.internal._broadcast_shapes`.  Our implementation could be removed once NumPy<1.20 is dropped and we check the performance of `numpy.broadcast_shapes`.